pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2208,6 +2236,86 @@ func (s *AWSCloud) EnsureLoadBalancer(apiService *api.Service, hosts []string, a
 		internalELB = true
 	}
 
+	// Some load balancer attributes are required, so defaults are set. These can be overridden by annotations.
+	loadBalancerAttributes := &elb.LoadBalancerAttributes{
+		AccessLog:              &elb.AccessLog{Enabled: aws.Bool(false)},
+		ConnectionDraining:     &elb.ConnectionDraining{Enabled: aws.Bool(false)},
+		ConnectionSettings:     &elb.ConnectionSettings{IdleTimeout: aws.Int64(60)},
+		CrossZoneLoadBalancing: &elb.CrossZoneLoadBalancing{Enabled: aws.Bool(false)},
+	}
+
+	// Determine if an access log emit interval has been specified
+	accessLogEmitIntervalAnnotation := annotations[ServiceAnnotationLoadBalancerAccessLogEmitInterval]
+	if accessLogEmitIntervalAnnotation != "" {
+		accessLogEmitInterval, err := strconv.ParseInt(accessLogEmitIntervalAnnotation, 10, 64)
+		if err != nil {
+			glog.Error("Error parsing access log emit interval; ignoring: ", err)
```

Comment:
We normally treat this as errors i.e. return fmt.Errorf, not warnings.  Users typically won't see warnings, and we don't want to silently not do something they told us to do.

It's nice to do something like:

`return nil, fmt.Errorf("error parsing service annotation: %s=%s", ServiceAnnotationLoadBalancerAccessLogEmitInterval, accessLogEmitIntervalAnnotation)`

Normally we include the error, but in this case I don't know if there's any point (given it can be trivially determined from the value).

It's fine _not_ to do this now - we apparently have an issue with bubbling up of events anyway, so I'm going to be looking at this anyway.